[DOCS] Doc privs needed to import rules with actions (#1762)
elastic/security-docs: docs/detections/rules-ui-manage.asciidoc

+
NOTE: You cannot export Elastic prebuilt rules.
. To import rules:
+

> NOTE: Your role needs at least `Read` privileges for the `Action and Connectors` feature to import rules with actions. If you are importing rules without actions, no `Action and Connectors` feature privileges are required. Refer to <<enable-detections-ui>> for more information.

Reviewer: ```suggestion
NOTE: Your role needs at least `Read` privileges for the `Action and Connectors` feature to import rules with actions. If you're importing rules without actions, `Action and Connectors` feature privileges are not required. Refer to <<enable-detections-ui>> for more information.
```
